package.json

```diff
@@ -59,7 +59,7 @@
     "webpack-cli": "^4.10.0"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.3.0"
+    "@eppo/js-client-sdk-common": "https://github.com/Eppo-exp/js-sdk-common.git#0bd1de8d06426d91006cfda5070554001ff1184e"
```

Comment:
this will eventually point to the next release version of `js-client-sdk-common` once that's published